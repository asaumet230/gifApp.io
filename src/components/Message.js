import React from 'react'

const Message = () => {


  return (
    <div className='mensaje'>
        <h2>Ups!</h2>
        <p> No hay resultados  disponible pero si tienes historial has click en este. </p>
    </div>
  )
}

export default Message