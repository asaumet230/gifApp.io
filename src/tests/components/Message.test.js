import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

import Message from '../../components/Message';


describe('Pruebas componente <Message />', () => { 

    let wrapper = shallow( <Message /> );

    test('Componente <Message /> debe renderizarse correctamente con un snapShot', () => { 
        expect( wrapper ).toMatchSnapshot();
    });

    test('Componente <Message /> debe tener el className= "mensaje"', () => { 
        
        const div = wrapper.find('div');

        expect(div.hasClass('mensaje')).toBe(true);

     });



});