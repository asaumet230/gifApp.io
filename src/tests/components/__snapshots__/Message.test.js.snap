// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas componente <Message /> Componente <Message /> debe renderizarse correctamente con un snapShot 1`] = `
<div
  className="mensaje"
>
  <h2>
    Ups!
  </h2>
  <p>
     No hay resultados  disponible pero si tienes historial has click en este. 
  </p>
</div>
`;
